maxscale/Chart.yaml

```
@@ -1,7 +1,7 @@
 ---
 apiVersion: v1
 name: maxscale
-version: 1.1.1
+version: 1.1.2
```

The new values justify a minor release
```suggestion
version: 1.2.0
```

---

Resolved